tests/cli_tests.py

```diff
@@ -59,8 +59,10 @@ def test_import_controlled_list(self):
                 source=input_file,
                 stdout=output,
             )
-        list_pk = "e962bdaf-8243-4fbb-bd43-39bc1f54c168"
-        self.assertTrue(List.objects.filter(pk=list_pk).exists())
+
+        self.assertTrue(List.objects.all().count() == 2)
+        self.assertTrue(ListItem.objects.all().count() == 10)
+        self.assertTrue(ListItemValue.objects.all().count() == 21)
```

Comment:
Nice, this is a lot better.

FYI, I try to avoid assertTrue/assertFalse. When it fails, all the information you get is `True is not False`. If you assertEqual, you get output like 1 != 2.